schemas/rum/view-schema.json

```diff
@@ -31,6 +31,18 @@
               "minimum": 0,
               "readOnly": true
             },
+            "network_settled_time": {
+              "type": "integer",
+              "description": "Duration in ns from the moment the initial network requests in current view started until they settled",
+              "minimum": 0,
+              "readOnly": true
+            },
+            "interaction_to_next_view_time": {
+              "type": "integer",
+              "description": "Duration in ns to from the last interaction on previous view to the moment the next view is displayed",
```

Comment:
> [!NOTE]
> The metric relates to the current view, not the next (future) one. 

```suggestion
              "description": "Duration in ns to from the last interaction on previous view to the moment the current view was displayed",
```

Reply:
yeah...for me `next` is actually `current` in the context. But if you think `current` is better I can change it yes